auto_property_call.py

```diff
@@ -0,0 +1,46 @@
+"""This script automatically wraps get, set, del methods in property."""
+
+
+class Meta(type):
+    """Metaclass that augments subclass instantiation."""
+
+    def __call__(cls, *args, **kwargs):
+        """Override the __call__ method."""
+        new = cls.__new__(cls, *args, **kwargs)
+        if isinstance(new, cls):
+            new.__init__(*args, **kwargs)
+        x_methods = [getattr(cls, attr) for attr in vars(cls)
+                     for substr in ['get_x', 'set_x', 'del_x']
+                     if substr in attr]
+        y_methods = [getattr(cls, attr) for attr in vars(cls)
+                     for substr in ['get_y', 'set_y', 'del_y']
+                     if substr in attr]
+        cls.x = property(*tuple(method for method in x_methods))
```

Comment:
`tuple` can be omitted here
`cls.x = property(*(method for method in x_methods)